contracts/src/core/EigenDARegistryCoordinator.sol

```diff
@@ -940,33 +804,13 @@ contract EigenDARegistryCoordinator is
         return registries.length;
     }
 
-    /**
-     * @notice Public function for the the churnApprover signature hash calculation when operators are being kicked from quorums
-     * @param registeringOperatorId The id of the registering operator
-     * @param operatorKickParams The parameters needed to kick the operator from the quorums that have reached their caps
-     * @param salt The salt to use for the churnApprover's signature
-     * @param expiry The desired expiry time of the churnApprover's signature
-     */
-    function calculateOperatorChurnApprovalDigestHash(
-        address registeringOperator,
-        bytes32 registeringOperatorId,
-        OperatorKickParam[] memory operatorKickParams,
-        bytes32 salt,
-        uint256 expiry
-    ) public view returns (bytes32) {
-        // calculate the digest hash
-        return _hashTypedDataV4(
-            keccak256(
-                abi.encode(
-                    OPERATOR_CHURN_APPROVAL_TYPEHASH,
-                    registeringOperator,
-                    registeringOperatorId,
-                    operatorKickParams,
-                    salt,
-                    expiry
-                )
-            )
-        );
+    /// @notice Deprecated function.
```

Comment:
nit: think its good in general to add info as to why its deprecated and when we plan to remove it totally.

Reply:
also makes sure someone doesn't just come in and delete it accidentaly before we have finished the offchain migration period.